inst/extdata/plumber.R

```diff
@@ -1,5 +1,7 @@
 # Packages ----
 library(plumber)
+library(future)
+plan(multisession, workers = 25)
```

Comment:
What does the `workers` parameter do and why 25?

Reply:
`workers` is the number of parallel processes to use. See https://future.futureverse.org/reference/multisession.html